SampleProject/AndOrClauseViewController.swift

```diff
@@ -333,7 +333,7 @@ class AndOrClauseViewController: KiiBaseTableViewController, UIPickerViewDataSou
                 delegate!.saveClause(newClause)
             }
         }
-        self.navigationController?.popViewController(animated: true)
+        _ = self.navigationController?.popViewController(animated: true)
```

Comment:
This not good warning removal.
`self.navigationController!.popViewController(animated: true)` 
is better approach since if navigation controller is nil, there must be incorrect usage somewhere. Forced unwrapping  is more appropriate than `_` literal